src/pages/login.js

```diff
@@ -13,16 +13,21 @@ function Login() {
     const pass = data.get("password");
 
     if (mail && pass) {
-      await post(`auth/login`, {
-        email: mail,
-        password: pass,
-      }).then((response) => {
+      try {
+        const response = await POST(`auth/login`, {
```

Comment:
Is the POST not a promise? Why do you need to catch an error?

Reply:
Because we need to know when the login fails due to invalid credentials to not validate the session.

Reply:
Right, but if it's a promise shouldn't that be possible to do with [`catch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch)?